pipeline_plugins/components/collections/sites/open/nodeman/create_task/v4_0.py

```diff
@@ -96,7 +96,17 @@ def execute(self, data, parent_data):
                 ap_id = host["nodeman_ap_id"]
                 auth_type = host["auth_type"]
                 auth_key = host["auth_key"]
-                inner_ip_list = get_ip_by_regex(host.get("inner_ip"))
+
+                if settings.ENABLE_IPV6:
```

Comment:
use_inner_ip = True if host.get("inner_ip") else False
inner_ip_list = self.get_ip_list(host.get("inner_ip", "") if use_inner_ip or not settings.ENABLE_IPV6 else host.get("inner_ipv6", ""))